src/core/eval.ml

```diff
@@ -474,19 +447,44 @@ let unfold_sym : sym -> term -> term =
   match !(s.sym_rules) with
   | [] -> fun t -> t
   | _ ->
-      let dt = !(s.sym_dtree) in
+      let c = cfg_of_ctx [] true and dt = !(s.sym_dtree) in
```

Comment:
The `and` is not needed here, if I'm not mistaken.

Reply:
What do you mean? This just means that the two can expressions be computed in parallel.

Reply:
Yes, but it introduces unneeded mutual dependencies between definitions, and doesn't speed up anything.

Reply:
No, its a let not a let rec.

Reply:
Ah right!